Analysis/source/ProcessAnaTuple.cxx

```diff
@@ -101,9 +65,48 @@ class ProcessAnaTuple : public EventAnalyzerCore {
                                             false, bkg_names, unc_collection);
         const EventSubCategorySet& subCategories = sub_categories_to_process;
 
-        std::cout << "\tCreating histograms..." << std::endl;
-        ProduceHistograms(anaDataCollection);
+        std::vector<std::string> unc_sources = SplitValueList(args.unc_sources_groups(),false, ",", true);
+        for (const auto& unc_source : unc_sources){
+            std::string input_file = args.input()+unc_source+".root";
+            std::cout<< "\n \t input file " << input_file << std::endl;
+            bbtautau::AnaTupleReader tupleReader(input_file, args.channel(), activeVariables, args.input_friends(), eventTagger,
```

Comment:
friends will also change depending on the input. Better to have a pattern in the name that corresponds to unc_source placement.